src/coretransport.cpp

```diff
@@ -232,7 +232,20 @@ void CoreTransport::TransportConnectHelper()
 
     WriteParameterFilesOut out;
     WriteParameterFilesIn in;
-    in.requestingUrlProxyWithoutTunnel = RequestingUrlProxyWithoutTunnel();
+    bool requestingUrlProxyWithoutTunnel = RequestingUrlProxyWithoutTunnel();
+    in.requestingUrlProxyWithoutTunnel = requestingUrlProxyWithoutTunnel;
+    if (requestingUrlProxyWithoutTunnel) {
+        // RequireUrlProxyWithoutTunnel mode has a distinct config file so that

```

Comment:
Should this be `RequestingUrlProxyWithoutTunnel` rather than `RequireUrlProxyWithoutTunnel`?